core/build.gradle.kts

```diff
@@ -66,6 +66,10 @@ configurations.api {
     // This is still experimental - additionally, it could only really benefit standalone
     exclude(group = "io.netty.incubator", module = "netty-incubator-transport-native-io_uring")
 }
+java {
+    sourceCompatibility = JavaVersion.VERSION_16
+    targetCompatibility = JavaVersion.VERSION_16
+}
```

Comment:
Was this added because of the current indra issue? This should be removed

Reply:
yeee i'll yeet it